### PR TITLE
Feature/tension chart

### PR DESCRIPTION
**Pull request by Xristof23**
Log:

- write new Component `TensionChart` using `plotly`library
- add logic to convert tension dataa from emotionEntries state to be usable for chart
- render TensionChart component on index.js
- add button to show/hide Chart
- add fitting styling to TensionChart

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -78,61 +103,102 @@ export default function HomePage({ onAddEmotionEntry, handleToggleTooltip }) {
...
+
+  const xValues = emotionEntries
+    .filter((entry) => currentShortDate === entry.isoDate?.slice(0, 10))
+    .sort(compare)
+    .map((entry) => entry.timeAndDate.slice(-5));
+
+  const yValues = emotionEntries
+    .filter((entry) => currentShortDate === entry.isoDate?.slice(0, 10))
+    .sort(compare)
+    .map((entry) => entry.tensionLevel);
```
Comment (F-Kirchhoff) [inline pages/index.js:126]: here you are performing double the work. Please filter and sort first and then split the values into xValues and yValues:

```js
const data = emotionEntries
    .filter((entry) => currentShortDate === entry.isoDate?.slice(0, 10))
    .sort(compare)
const xValues = data.map(entry => entry.timeAndDate.slice(-5))
const yValues = data.map((entry) => entry.tensionLevel);
```

File: pages/index.js [hunk 2]
```diff
@@ -78,61 +103,102 @@ export default function HomePage({ onAddEmotionEntry, handleToggleTooltip }) {
...
+              <StyledMessage>Your entry was successfully saved!</StyledMessage>
+              <StyledButtonWrapper>
+                <StyledBackButton
+                  type="reset"
+                  value={"Done"}
+                  onClick={() => {
+                    setIsFormSubmitted(!isFormSubmitted);
+                    setTension("0");
+                  }}
+                ></StyledBackButton>
```
Comment (F-Kirchhoff) [inline pages/index.js:176]: please use self closing tags if possible (whenever you don't pass children to a component)

**Review by F-Kirchhoff — CHANGES_REQUESTED** [review]
Hey Emotions Team,
very nice work! There is only one small issue where you can save a bit of computing time by reusing data. Please take a look!

File: pages/index.js (hunk 1 above)
Comment (Xristof23) [reply pages/index.js]: Hi, thanks for the review. Yeah, I had the feeling that bit of my code could have been better ... but the solution I had in mind was not as short as yours so I did not change it. But changed it now thanks to your suggestion.

File: pages/index.js (hunk 2 above)
Comment (Xristof23) [reply pages/index.js]: Done

**Review by F-Kirchhoff — APPROVED** [review]
lgtm

**Pull request merged**